lib/ui/tiles/subscribe_button.dart

```diff
@@ -38,12 +38,12 @@ class _SubscribeButtonState extends State<SubscribeButton> {
 
   @override
   Widget build(BuildContext context) {
-    return IconButton(
```

Comment:
Why is this change needed?

Reply:
Because Icon Button takes a liitle space apart from the icon and thats why the Subscribe button is not shifted to the extreme right. Thats why I changed to Gesture detector.

Reply:
Alright.